Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -25,6 +25,10 @@
 
 #pragma once
 
+#if USE(CF)
```

Comment:
I'm pretty sure nobody is going to like this, but I couldn't find another place to put this that worked for all parent `#include`s. It really should be in `"GeneratedSerializers.h"` but I don't know how to make that happen.

Reply:
Yeah, this isn't good.  We also don't want it in GeneratedSerializers.h.  All we want in GeneratedSerializers.h is "typedef struct __SecTrust *SecTrustRef;"

Reply:
`headers: <Security/Security.h>` in the serialization.in should help?

Reply:
Unfortunately, it does not.

Reply:
On the plus side, the next 3 types I plan to tackle are the `Sec*Ref` types, which all will fall under this one header.

Reply:
We don't want to include Security.h here.  That is too broad, which slows down build times, and "typedef struct __SecTrust *SecTrustRef;" will do just fine.